.github/workflows/ui-test.yml

```diff
@@ -57,44 +57,12 @@ jobs:
       - name: Composer install
         run: composer install
 
-      - name: Backup
+      - name: Display new interface
         run: |
-          docker exec -t prestashop php modules/autoupgrade/bin/console backup:create admin-dev
-
-      - name: Download local ZIP and XML for local channel
-        if: matrix.UPGRADE_CHANNEL == 'local' && matrix.PS_VERSION_END != '9.0.0'
-        run: |
-          docker exec -t prestashop curl --fail -L https://github.com/PrestaShop/zip-archives/raw/main/prestashop_${{ matrix.PS_VERSION_END }}.zip -o admin-dev/autoupgrade/download/prestashop_${{ matrix.PS_VERSION_END }}.zip
-          docker exec -t prestashop curl --fail -L https://api.prestashop.com/xml/md5/${{ matrix.PS_VERSION_END }}.xml -o admin-dev/autoupgrade/download/prestashop_${{ matrix.PS_VERSION_END }}.xml
-
-      - name: Download local ZIP and XML for local channel in 9.0.0
-        if: matrix.PS_VERSION_END == '9.0.0'
-        run: |
-          docker exec -t prestashop curl --fail -L https://storage.googleapis.com/prestashop-core-nightly/nightly_9.0.x.zip -o admin-dev/autoupgrade/download/prestashop_${{ matrix.PS_VERSION_END }}.zip
-          docker exec -t prestashop curl --fail -L https://storage.googleapis.com/prestashop-core-nightly/nightly_9.0.x.xml -o admin-dev/autoupgrade/download/prestashop_${{ matrix.PS_VERSION_END }}.xml
-
-      - name: Write configuration file
-        run: |
-          docker exec -t prestashop sh -c "echo '{
-            \"channel\":\"${{ matrix.UPGRADE_CHANNEL }}\",
-            \"archive_zip\":\"prestashop_${{ matrix.PS_VERSION_END }}.zip\",
-            \"archive_xml\":\"prestashop_${{ matrix.PS_VERSION_END }}.xml\",
-            \"PS_AUTOUP_CUSTOM_MOD_DESACT\":\"true\",
-            \"PS_AUTOUP_CHANGE_DEFAULT_THEME\":\"false\",
-            \"PS_AUTOUP_KEEP_IMAGES\":\"true\",
-            \"PS_DISABLE_OVERRIDES\":\"true\"
-          }' > modules/autoupgrade/config.json"
-
-      - name: Update
-        run: |
-          docker exec -t prestashop php modules/autoupgrade/bin/console update:start --config-file-path=modules/autoupgrade/config.json --channel=${{ matrix.UPGRADE_CHANNEL }} admin-dev
-          docker exec -t prestashop chmod 777 -R /var/www/html/var
-
-      - name: Uninstall welcome module
-        if: matrix.PS_VERSION_START == '1.7.0.6' || matrix.PS_VERSION_START == '1.7.1.0' || matrix.PS_VERSION_START == '1.7.1.1' || matrix.PS_VERSION_START == '1.7.1.2'
-        run: |
-          docker exec -t prestashop php bin/console prestashop:module disable welcome
-          docker exec -t prestashop chmod 777 -R /var/www/html/var
+          docker exec -t prestashop bash -c 'apt-get update'
+          docker exec -t prestashop bash -c 'curl -sL https://deb.nodesource.com/setup_20.x | bash -'
+          docker exec -t prestashop bash -c 'apt-get install -y nodejs'
+          docker exec -t prestashop bash -c 'cd /var/www/html/modules/autoupgrade/_dev && npm install && npm run vite:build'
```

Comment:
Whats the reason to build the UI inside a docker container, where the github action already provides the tools to do so?